Formula/open-mpi.rb

```diff
@@ -3,6 +3,11 @@ class OpenMpi < Formula
   homepage "https://www.open-mpi.org/"
   url "https://www.open-mpi.org/software/ompi/v2.0/downloads/openmpi-2.0.1.tar.bz2"
   sha256 "fed74f4ae619b7ebcc18150bb5bdb65e273e14a8c094e78a3fea0df59b9ff8ff"
+  revision 1
+  patch do
+    url "https://github.com/open-mpi/ompi/pull/2135.patch"
```

Comment:
Can you add a brief comment describing what the patch is needed for? Thanks for submitting a PR to both projects, though!